portfolio/src/main/webapp/script.js

```diff
@@ -14,15 +14,34 @@
 
 
 /**
- * Add a random photo to the page.
+ * Adds a random fact to the page.
  **/
  function addRandomFact() {
      const facts = ['I am bilingual', 'I love plants', 'I have purple hair', 'My favorite color is blue'];
      //Choose a random fact
      const randomFact = facts[Math.floor(Math.random() * facts.length)];
      
-     //Add random fact to the page
+     //Add it to the page
```

Comment:
the reason i changed it was because on the comment of the method i already mentioned that is adding a random fact so i didn't want to sound repetitive. 
Do you think it's better to revert that change or just delete the method comment altogether?

Reply:
Either is fine. I don't feel any strong preference between the 2 options.